proteus/mprans/SW2DCV.h

```diff
@@ -3269,7 +3248,7 @@ namespace proteus
       //FOR FRICTION//
       double n2 = std::pow(mannings,2.);
       double gamma=4./3;
-      double xi=2.;
+      double xi=10.;
```

Comment:
should put a comment on what xi is

Reply:
The factor xi alleviates the cfl restriction. I need to see the behavior after incorporating the local bounds recently incorporated into the paper.

Reply:
Got it. What I mean is that you should put a comment in the code if it's something that is subject to change and not (yet) set by the theory or numerical experiments.